ext/travis/test

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ "$TRAVIS_OS_NAME" = linux ]; then
+    uid="$(id -u)"
+    gid="$(id -g)"
+    tinyfs="$(pwd)/tinyfs"
+
+    sudo -i dd if=/dev/zero of=tinyfs.img bs=10MB count=1
+    sudo -i chmod 600 tinyfs.img
+    sudo -i mke2fs -F tinyfs.img
+    sudo -i mkdir "$tinyfs"
+    sudo -i mount -oloop tinyfs.img "$tinyfs"
+    sudo -i chown "$uid:$gid" "$tinyfs"
```

Comment:
I don't know, I'm not familiar with the contents of these travis scripts; only that they exist in our repos 😄  I would take a peek at some of the existing travis scripts we have to see if they use `sudo` or not.